SPEC.md

```diff
@@ -70,6 +70,19 @@ The operations that the CNI plugin needs to support are:
     - **Extra arguments**, as defined above.
```

Comment:
For some reason github won't let me comment on the right line.  Anyway, can we add something to line 61 the "IP configuration assigned to each interface" result to explain that you may not get the IP information immediately.

Can we also describe what the expectations are about a GET?  Is the container runtime expected to periodically call GET throughout the lifecycle of the container?  (And how often?)

Can interfaces be added while the container is running?

Once an interface has an IP address... can it change?  If it has a v4 address, can it later get a v6 address?

Is there a way to know that the network plugin is "done" and that the config is final and doesn't need to be polled any more?